hatchet_sdk/workflows_pb2_grpc.py

```diff
@@ -2,7 +2,7 @@
 """Client and server classes corresponding to protobuf-defined services."""
 import grpc
 
-from . import workflows_pb2 as workflows__pb2
+import workflows_pb2 as workflows__pb2
```

Comment:
im pretty sure we need the `from . `, we should fix this with a sed pre commit hook

Reply:
do you have any idea why this happens? since I'm using the same thing as from hatchet:main but for some reason this formats slightly different